boards/nucleo-f103/include/board.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2015 TriaGnoSys GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_nucleo-f103 Nucleo-F103
+ * @ingroup     boards
+ * @brief       Board specific files for the nucleo-f103 board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the nucleo-f103 board
+ *
+ * @author      Víctor Ariño <victor.arino@triagnosys.com>
+ */
+
+#ifndef BOARD_H_
+#define BOARD_H_
+
+#include <stdint.h>
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Define the nominal CPU core clock in this board
+ */
+#define F_CPU               CLOCK_CORECLOCK
+
+/**
+ * @name Define the UART to be used as stdio and its baudrate
+ * @{
+ */
+#define STDIO               UART_1
+#define STDIO_BAUDRATE      (9600U)
```

Comment:
Any reason for not using the default `115200` here? The boards is clocked with 72MHz, so a higher baudrate should not be an issue performance wise...